volatility/framework/automagic/mac.py

```diff
@@ -214,7 +214,8 @@ def virtual_to_physical_address(cls, addr: int) -> int:
         return addr - 0xffffff8000000000
 
     @classmethod
-    def files_descriptors_for_process(cls, config: interfaces.configuration.HierarchicalDict,
+    def files_descriptors_for_process(cls, 
+                                      symbol_table_name : str,
```

Comment:
Typically we pass the context as the first argument, but it's not a show stopper...

Reply:
Also, good to have docstrings for a classmethod that's exposed as an API by this plugin.

Reply:
Any thoughts on putting `context` first?  I'd quite like people to consistently be able to intuit which arguments go in which order, and so far context has always come first if it's needed.  If there's a good reason not to, I'm open to it, but otherwise it'd make a nicer more consistent API to have it come first please...